assets/src/containers/FilesAccessed.js

```diff
@@ -189,9 +200,23 @@ function FilesAccessed (props) {
               onWeekChange={onWeekChangeHandler}
             /> : ''}
             <div className={classes.formController}>
-              <p>File accessed from
+              <FormControl className={classes.formControl}>
```

Comment:
From the UI I feel the types of data need not be a dropdown but I think a cluster of checkboxes would be appropriate in case if we need to have more data sources like `kaltura`/ `runstone` etc. I think we need to check with Jennifer love how the UI should look like and function

Reply:
Yeah, I think that would be pretty neat. It looks like there's a few multiselects out there for react
https://jedwatson.github.io/react-select/
https://github.com/Khan/react-multi-select

Reply:
React being using Material-UI components. we could leverage this https://material-ui.com/components/checkboxes/